src/main/resources/static/css/widget.css

```diff
@@ -167,6 +167,9 @@ h2 {
 }
 
 /*INCOMPATIBLE PAGE*/
+.incompatible {
+    margin-top: 65px;
```

Comment:
Good catch! Actually, 65 is 1 px too much, but I think that's fine. 😉

Reply:
yes I did initially 64px, but somehow in my 1080p external screen, it looks like the top edge is cut off, so that's why I added 65px 😅